exporters/otlp/otlp_metric_test.go

```diff
@@ -767,12 +767,7 @@ func TestStatelessExportKind(t *testing.T) {
 
 // What works single-threaded should work multi-threaded
 func runMetricExportTests(t *testing.T, opts []ExporterOption, rs []record, expected []metricpb.ResourceMetrics) {
```

Comment:
This outside function is likely not needed anymore.

Reply:
Yeah, I think I dropped it somewhere later in my branch. But anyway, I'll drop it now.